webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/batch/AsyncDeploymentActor.scala

```diff
@@ -67,6 +68,30 @@ final case class AutomaticStartDeployment(modId: ModificationId, actor:EventActo
 //actor: the actor who asked for the deployment
 final case class ManualStartDeployment(modId: ModificationId, actor:EventActor, reason:String) extends HashcodeCaching with StartDeploymentMessage
 
+
+sealed  trait GenerationPolicy
```

Comment:
I find GenerationPolicy extremely misleading, my brain absolutly wants to read "policy generation" and interpret it like that. 
Pehaps use: `GenerationTriggerEvent` (all, onlyManual, none) ?